.gitignore

```diff
@@ -5,3 +5,5 @@ node_modules/
 dist/
 
 globalConfig.json
+
+.idea
```

Comment:
I'm fine with including this here but in general things like `.idea` or `.vscode` should be placed in your global  gitignore on your machine